tasks/vmaas_sync/package_sync_test.go

```diff
@@ -15,9 +16,36 @@ func TestPkgListSyncPackages(t *testing.T) {
 	core.SetupTestEnvironment()
 	configure()
 
+	var oldNameCount int
+	var oldPkgCount int
+	var newNameCount int
+	var newPkgCount int
+	var pkgNextval int
+	var nameNextval int
+	var pkgCurrval int
+	var nameCurrval int
```

Comment:
Some convenience of Go :slightly_smiling_face: 
```
	var (
		oldNameCnt  int
		oldPkgCnt   int
                ...
	)
```

Reply:
I'd prefer
```
var oldNameCount, oldPkgCount, newNameCount, newPkgCount int
var pkgNextval, nameNextval, pkgCurrval, nameCurrval int
```
to shorten the text of the function (so it fits on the screen).

Reply:
done